_events/2024-03-23-a-sample-event.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: event
+title: "OLS: a training and mentoring program for the open-research-curious"
+description: "Detailed event description"
+date: 2024-03-23
+time: "10:00 AM WAT"
+duration: 2 hours
+location: "Online"
+speakers:
+  - name: "Y.Y"
+  - affiliation: "Open Life Science"
+registration: ""
```

Comment:
I would remove it. If it has a registration, it could be in the text below instead of the metadata